west.yml

```diff
@@ -25,6 +25,8 @@ manifest:
   remotes:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
+    - name: pabigot
+      url-base: https://github.com/pabigot
```

Comment:
I assume this is until we host littlefs module in Zephyr project?

Reply:
Yes, @carles is creating that today.